airflow/plugins_manager.py

```diff
@@ -208,9 +287,9 @@ def make_module(name, objects):
 
 for p in plugins:
     operators_modules.append(
-        make_module('airflow.operators.' + p.name, p.operators + p.sensors))
+        make_deprecated_module('airflow.operators.' + p.name, p.operators + p.sensors, p))
     sensors_modules.append(
-        make_module('airflow.sensors.' + p.name, p.sensors)
+        make_deprecated_module('airflow.sensors.' + p.name, p.sensors, p)
     )
     hooks_modules.append(make_module('airflow.hooks.' + p.name, p.hooks))
```

Comment:
Should we deprecate adding Hooks and Executors too??

Reply:
Separate ticket for hooks (cos it maybe makes sense to register them, if not import them from hooks) #9506 and #9507